### PR TITLE
feat: multiple partners for single owner account fees collection balance

### PR DESCRIPTION
**Pull request by sergio-jara**
https://rsklabs.atlassian.net/browse/RNS-104?atlOrigin=eyJpIjoiODcxNTk3YjNiNDc4NDcxODhiMGNkNWZmYmQzOTA2ZTgiLCJwIjoiaiJ9

### CODE REVIEW

File: test/FeeManager.spec.ts [hunk 1]
```diff
@@ -184,25 +195,31 @@ describe('Fee Manager', () => {
...
+      ).eventually.fulfilled;
     });
 
     it('should withdraw successfully', async () => {
       try {
-        await expect(feeManager.connect(partner).withdraw()).to.not.be.reverted;
-        expect(await feeManager.balances(partner.address)).to.be.equals(
+        await expect(feeManager.connect(partnerOwnerAccount).withdraw()).to
+          .eventually.fulfilled;
+        expect(await feeManager.getBalance(partner.address)).to.be.equals(
```
Comment (OnyekaIjeh) [inline test/FeeManager.spec.ts:222]: Should be checking the balance for `partnerOwnerAccount`

File: contracts/FeeManager/FeeManager.sol [hunk 2]
```diff
@@ -73,4 +72,14 @@ contract FeeManager is IFeeManager, Ownable {
...
         return _partnerManager.getPartnerConfiguration(partner);
     }
+
+    function _getPartnerOwnerAccount(
+        address partner
+    ) private view returns (address) {
+        return _partnerManager.getPartnerOwnerAccount(partner);
+    }
+
+    function getBalance(address partner) external view returns (uint256) {
```
Comment (OnyekaIjeh) [inline contracts/FeeManager/FeeManager.sol:82]: aShould we consider not making this method public if we're going to make the `balances` not public either way?
Not sure there is much difference between a public balances field and a public method for getting the balance with an address parameter

**Review by OnyekaIjeh — COMMENTED** [review]
Left a few comments

File: test/FeeManager.spec.ts (hunk 1 above)
Comment (sergio-jara) [reply test/FeeManager.spec.ts:222]: yep

**Review by OnyekaIjeh — APPROVED** [review]
Looks good so far.

**Pull request merged**